### ISSUE TITLE
As a user, i don't want that the card I'm currently looking can disappear

### ISSUE THREAD

**Issue by vincentbarbesant** [opfab/operatorfabric-core#91]
Context : 
- The filter is set to not show the notification cards (default behavior)
- I'm currently looking at a card (the content of this card is displayed on the screen). Since this card is visible, it is not a notification card.

Event : the card is updated (by a process, or following an action I've made in the card), and is now is now a notification card.

What should happen ? In term of UX, I would say that i don't want the card to disappear immediately, because I'm working on it. It would be strange that the UI I'm currently using suddenly disappear. I would suggest the card disappear only after I select another card.

**Comment by AlexGuironnetRTE** [comment]
Moved to [OC-688](https://opfab.atlassian.net/browse/OC-688)
